src/Symfony/Bridge/Twig/Extension/CodeExtension.php

```diff
@@ -202,6 +202,10 @@ public function formatFile($file, $line, $text = null)
      */
     public function getFileLink($file, $line)
     {
+        //If on Windows, we must escape $file if it contains backslash char
+        if('\\' === DIRECTORY_SEPARATOR && FALSE !== strpos($file, '\\')) {
+            $file = addslashes($file);
```

Comment:
As said before, this is a nogo as most cases don't want such escaping. You will probably break it for all other users.